pkg/framework/host.go

```diff
@@ -48,8 +49,9 @@ type Host struct {
 	logger     micrologger.Logger
 	filelogger *filelogger.FileLogger
 
+	extClient            *apiextensionsclient.Clientset
 	g8sClient            *versioned.Clientset
-	k8sClient            kubernetes.Interface
+	k8sClient            *kubernetes.Clientset
```

Comment:
Will this be breaking for k8sClient? Or is this fine?

Reply:
This should be fine. Interface is still returned outside. I just aligned stuff.